test/test_mask-p.js

```diff
@@ -0,0 +1,11 @@
+var assert = require("assert");
+var dateFormat = require("../lib/dateformat");
+
+describe("Mask: 'p'", function () {
+  it("should get timezone for any date as smoething like [+-]XX:XX", function (done) {
```

Comment:
@fboes Spelling error `smoething` should be `something`

Reply:
I fixed that spelling mistake. It's one of my favourite typos early in the morning without a coffee. :)